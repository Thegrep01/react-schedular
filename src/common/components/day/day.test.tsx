import React from 'react';
import renderer from 'react-test-renderer';
import Day from './day';

describe('render day', () => {
    it('render day', () => {
        const clock = renderer.create(<Day day='Monday'></Day>).toJSON();
        expect(clock).toMatchSnapshot();
    });
});
