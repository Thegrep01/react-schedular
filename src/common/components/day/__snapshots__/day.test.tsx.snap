// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`render day render day 1`] = `
<div
  className="MuiBox-root MuiBox-root-2"
>
  <h4
    className="MuiTypography-root MuiTypography-h4"
  >
    Monday
  </h4>
  <ul
    className="MuiList-root list MuiList-padding"
  >
    <li
      className="MuiListItem-root MuiListItem-gutters"
      disabled={false}
    >
      <div
        className="MuiPaper-root MuiPaper-elevation1 MuiCard-root card MuiPaper-rounded"
      >
        <div
          className="MuiCardContent-root"
        >
          <div
            className="MuiBox-root MuiBox-root-77"
          >
            <p
              className="MuiTypography-root title MuiTypography-body1 MuiTypography-colorTextSecondary MuiTypography-gutterBottom"
            >
              13:00
            </p>
            <p
              className="MuiTypography-root title MuiTypography-body1 MuiTypography-colorTextSecondary MuiTypography-gutterBottom"
            >
              6/52
            </p>
          </div>
          <h2
            className="MuiTypography-root MuiTypography-h5"
          >
            Lorem ipsum dolor
          </h2>
          <p
            className="MuiTypography-root MuiTypography-body1 MuiTypography-colorTextSecondary"
          >
            Lorem ipsum dolor
          </p>
        </div>
      </div>
    </li>
    <li
      className="MuiListItem-root MuiListItem-gutters"
      disabled={false}
    >
      <div
        className="MuiPaper-root MuiPaper-elevation1 MuiCard-root card MuiPaper-rounded"
      >
        <div
          className="MuiCardContent-root"
        >
          <div
            className="MuiBox-root MuiBox-root-78"
          >
            <p
              className="MuiTypography-root title MuiTypography-body1 MuiTypography-colorTextSecondary MuiTypography-gutterBottom"
            >
              13:00
            </p>
            <p
              className="MuiTypography-root title MuiTypography-body1 MuiTypography-colorTextSecondary MuiTypography-gutterBottom"
            >
              6/52
            </p>
          </div>
          <h2
            className="MuiTypography-root MuiTypography-h5"
          >
            Lorem ipsum dolor
          </h2>
          <p
            className="MuiTypography-root MuiTypography-body1 MuiTypography-colorTextSecondary"
          >
            Lorem ipsum dolor
          </p>
        </div>
      </div>
    </li>
    <li
      className="MuiListItem-root MuiListItem-gutters"
      disabled={false}
    >
      <div
        className="MuiPaper-root MuiPaper-elevation1 MuiCard-root card MuiPaper-rounded"
      >
        <div
          className="MuiCardContent-root"
        >
          <div
            className="MuiBox-root MuiBox-root-79"
          >
            <p
              className="MuiTypography-root title MuiTypography-body1 MuiTypography-colorTextSecondary MuiTypography-gutterBottom"
            >
              13:00
            </p>
            <p
              className="MuiTypography-root title MuiTypography-body1 MuiTypography-colorTextSecondary MuiTypography-gutterBottom"
            >
              6/52
            </p>
          </div>
          <h2
            className="MuiTypography-root MuiTypography-h5"
          >
            Lorem ipsum dolor
          </h2>
          <p
            className="MuiTypography-root MuiTypography-body1 MuiTypography-colorTextSecondary"
          >
            Lorem ipsum dolor
          </p>
        </div>
      </div>
    </li>
    <li
      className="MuiListItem-root MuiListItem-gutters"
      disabled={false}
    >
      <div
        className="MuiPaper-root MuiPaper-elevation1 MuiCard-root card MuiPaper-rounded"
      >
        <div
          className="MuiCardContent-root"
        >
          <div
            className="MuiBox-root MuiBox-root-80"
          >
            <p
              className="MuiTypography-root title MuiTypography-body1 MuiTypography-colorTextSecondary MuiTypography-gutterBottom"
            >
              13:00
            </p>
            <p
              className="MuiTypography-root title MuiTypography-body1 MuiTypography-colorTextSecondary MuiTypography-gutterBottom"
            >
              6/52
            </p>
          </div>
          <h2
            className="MuiTypography-root MuiTypography-h5"
          >
            Lorem ipsum dolor
          </h2>
          <p
            className="MuiTypography-root MuiTypography-body1 MuiTypography-colorTextSecondary"
          >
            Lorem ipsum dolor
          </p>
        </div>
      </div>
    </li>
    <li
      className="MuiListItem-root MuiListItem-gutters"
      disabled={false}
    >
      <div
        className="MuiPaper-root MuiPaper-elevation1 MuiCard-root card MuiPaper-rounded"
      >
        <div
          className="MuiCardContent-root"
        >
          <div
            className="MuiBox-root MuiBox-root-81"
          >
            <p
              className="MuiTypography-root title MuiTypography-body1 MuiTypography-colorTextSecondary MuiTypography-gutterBottom"
            >
              13:00
            </p>
            <p
              className="MuiTypography-root title MuiTypography-body1 MuiTypography-colorTextSecondary MuiTypography-gutterBottom"
            >
              6/52
            </p>
          </div>
          <h2
            className="MuiTypography-root MuiTypography-h5"
          >
            Lorem ipsum dolor
          </h2>
          <p
            className="MuiTypography-root MuiTypography-body1 MuiTypography-colorTextSecondary"
          >
            Lorem ipsum dolor
          </p>
        </div>
      </div>
    </li>
    <li
      className="MuiListItem-root MuiListItem-gutters"
      disabled={false}
    >
      <div
        className="MuiPaper-root MuiPaper-elevation1 MuiCard-root card MuiPaper-rounded"
      >
        <div
          className="MuiCardContent-root"
        >
          <div
            className="MuiBox-root MuiBox-root-82"
          >
            <p
              className="MuiTypography-root title MuiTypography-body1 MuiTypography-colorTextSecondary MuiTypography-gutterBottom"
            >
              13:00
            </p>
            <p
              className="MuiTypography-root title MuiTypography-body1 MuiTypography-colorTextSecondary MuiTypography-gutterBottom"
            >
              6/52
            </p>
          </div>
          <h2
            className="MuiTypography-root MuiTypography-h5"
          >
            Lorem ipsum dolor
          </h2>
          <p
            className="MuiTypography-root MuiTypography-body1 MuiTypography-colorTextSecondary"
          >
            Lorem ipsum dolor
          </p>
        </div>
      </div>
    </li>
    <li
      className="MuiListItem-root MuiListItem-gutters"
      disabled={false}
    >
      <div
        className="MuiPaper-root MuiPaper-elevation1 MuiCard-root card MuiPaper-rounded"
      >
        <div
          className="MuiCardContent-root"
        >
          <div
            className="MuiBox-root MuiBox-root-83"
          >
            <p
              className="MuiTypography-root title MuiTypography-body1 MuiTypography-colorTextSecondary MuiTypography-gutterBottom"
            >
              13:00
            </p>
            <p
              className="MuiTypography-root title MuiTypography-body1 MuiTypography-colorTextSecondary MuiTypography-gutterBottom"
            >
              6/52
            </p>
          </div>
          <h2
            className="MuiTypography-root MuiTypography-h5"
          >
            Lorem ipsum dolor
          </h2>
          <p
            className="MuiTypography-root MuiTypography-body1 MuiTypography-colorTextSecondary"
          >
            Lorem ipsum dolor
          </p>
        </div>
      </div>
    </li>
    <li
      className="MuiListItem-root MuiListItem-gutters"
      disabled={false}
    >
      <div
        className="MuiPaper-root MuiPaper-elevation1 MuiCard-root card MuiPaper-rounded"
      >
        <div
          className="MuiCardContent-root"
        >
          <div
            className="MuiBox-root MuiBox-root-84"
          >
            <p
              className="MuiTypography-root title MuiTypography-body1 MuiTypography-colorTextSecondary MuiTypography-gutterBottom"
            >
              13:00
            </p>
            <p
              className="MuiTypography-root title MuiTypography-body1 MuiTypography-colorTextSecondary MuiTypography-gutterBottom"
            >
              6/52
            </p>
          </div>
          <h2
            className="MuiTypography-root MuiTypography-h5"
          >
            Lorem ipsum dolor
          </h2>
          <p
            className="MuiTypography-root MuiTypography-body1 MuiTypography-colorTextSecondary"
          >
            Lorem ipsum dolor
          </p>
        </div>
      </div>
    </li>
  </ul>
</div>
`;
